Supernova.Enum.Generators/Extensions/NamespaceSymbolExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+
+namespace Supernova.Enum.Generators.Extensions;
+
+/// <summary>
+/// Provides extension methods for <see cref="INamespaceSymbol"/> objects.
+/// </summary>
+public static class NamespaceSymbolExtensions
```

Comment:
It is better to have one extension, not several

Reply:
Yes, it is better to have a single extension file for a single target, especially if you only have a few methods.

But since this new file targets INamespaceSymbol and not ISymbol like the already existing one, the new file has a reason to exist.